css/style.css

```diff
@@ -0,0 +1,134 @@
+.body {
+  margin: 0 auto;
+  padding-left: 5px;
+}
+.first-field {
+  display: flex;
+  flex-wrap: wrap;
+}
+.svg-ff {
```

Comment:
What is `ff`?

Reply:
It were .class for SVGs from **f**irst **f**ield.